src/incubator/TextField/index.tsx

```diff
@@ -128,7 +128,7 @@ const TextField = (props: InternalTextFieldProps) => {
             )}
             {children || (
               <Input
-                placeholderTextColor={hidePlaceholder ? 'transparent' : Colors.$textNeutral}
+                placeholderTextColor={hidePlaceholder ? 'transparent' : placeholderTextColor}
```

Comment:
technically you're replacing the default value with undefined, no?

Reply:
No, I am changing the default for a non-floating placeholder from a `string` (set here to: `Colors.$textNeutral`) to an `object` (set in `default.ts`->`placeholderTextColorByState`); and this way I can set the `disabled` type to look like it's disabled: `Colors.$textDisabled`